src/audio/host.c

```diff
@@ -308,7 +309,6 @@ static void host_update_position(struct comp_dev *dev, uint32_t bytes)
 	struct comp_buffer *sink;
 	int ret;
 
-
```

Comment:
Please dont remove this new line here. Is not related to this patch. If you really want to remote it create another patch and explain why.

Reply:
@dbaluta  OK. Got it. I will fix it. I remove it because there are two extra lines here.